src/core/rewrite.ml

```diff
@@ -618,80 +618,81 @@ let rewrite : popt -> Proof.t -> rw_patt option -> term -> term =
 
   (* Construct the new goal and its type. *)
-  let goal_type = Appl(symb cfg.symb_P, new_term) in
+  let goal_type = Appl(Symb cfg.symb_P, new_term) in
   let goal_term = make_meta g_ctxt goal_type in
 
   (* Build the final term produced by the tactic, and check its type. *)
-  let eqind = symb cfg.symb_eqind in
+  let eqind = Symb cfg.symb_eqind in
   let term = add_args eqind [a; l; r; t; pred; goal_term] in
 
   (* Debugging data to the log. *)
   if !log_enabled then
     begin
       log_rewr "Rewriting with:";
-      log_rewr "  goal           = [%a]" pp g_type;
-      log_rewr "  equality proof = [%a]" pp t;
-      log_rewr "  equality type  = [%a]" pp t_type;
-      log_rewr "  equality LHS   = [%a]" pp l;
-      log_rewr "  equality RHS   = [%a]" pp r;
-      log_rewr "  pred           = [%a]" pp pred;
-      log_rewr "  new goal       = [%a]" pp goal_type;
-      log_rewr "  produced term  = [%a]" pp term;
+      log_rewr "  goal           = [%a]" pp_term g_type;
+      log_rewr "  equality proof = [%a]" pp_term t;
+      log_rewr "  equality type  = [%a]" pp_term t_type;
+      log_rewr "  equality LHS   = [%a]" pp_term l;
+      log_rewr "  equality RHS   = [%a]" pp_term r;
+      log_rewr "  pred           = [%a]" pp_term pred;
+      log_rewr "  new goal       = [%a]" pp_term goal_type;
+      log_rewr "  produced term  = [%a]" pp_term term;
     end;
 
   (* Return the proof-term. *)
   term
 
-(** [reflexivity ps] applies the reflexivity of equality on the focused  goal.
-    If successful, the corresponding proof term is returned. *)
-let reflexivity : popt -> Proof.t -> term = fun pos ps ->
+(** [reflexivity ss pos ps] applies the reflexivity of equality on the focused
+   goal. If successful, the corresponding proof term is returned. *)
+let reflexivity : Sig_state.t -> popt -> Proof.t -> term = fun ss pos ps ->
   (* Obtain the required symbols from the current signature. *)
-  let cfg = Proof.(get_eq_config pos ps.proof_builtins) in
+  let cfg = get_eq_config ss pos in
   (* Get the type of the focused goal. *)
   let _, g_type = Proof.focus_goal pos ps in
   (* Check that the type of [g] is of the form “P (eq a t t)”. *)
   let (a, l, r)  = get_eq_data pos cfg (Eval.whnf [] g_type) in
   if not (Eval.eq_modulo [] l r) then fatal pos "Cannot apply reflexivity.";
   (* Build the witness. *)
-  add_args (symb cfg.symb_refl) [a; l]
+  add_args (Symb cfg.symb_refl) [a; l]
 
-(** [symmetry ps] attempts to use symmetry of equality on the focused goal. If
-    successful,  a new goal is generated,  and the corresponding proof term is
-    returned. The proof of symmetry is built from the axioms of equality. *)
-let symmetry : popt -> Proof.t -> term = fun pos ps ->
+(** [symmetry ss pos ps] attempts to use symmetry of equality on the focused
+   goal. If successful, a new goal is generated, and the corresponding proof
+   term is returned. The proof of symmetry is built from the axioms of
+   equality. *)
```

Comment:
Comment not indented correctly.

Reply:
I will ignore these comments in the future.